client/reader/following/main.jsx

```diff
@@ -54,16 +53,14 @@ const FollowingStream = ( props ) => {
 	const placeholderText = getSearchPlaceholderText();
 	const { translate } = props;
 	const dispatch = useDispatch();
-	const voteBanner = <FollowingVoteBanner />;
 	const markAllAsSeen = ( feedsInfo ) => {
 		const { feedIds, feedUrls } = feedsInfo;
 		dispatch( requestMarkAllAsSeen( { identifier: SECTION_FOLLOWING, feedIds, feedUrls } ) );
 	};
-
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	return (
 		<Stream { ...props }>
-			{ voteBanner ? voteBanner : <FollowingIntro /> }
```

Comment:
This logic didn't work, because `voteBanner` didn't contain the rendered output of the component.